ROMFS/px4fmu_common/init.d/airframes/13015_jiyi_batman

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# @name JIYIRobot Batman
+#
+# @type VTOL Tiltrotor
+# @class VTOL
+#
+# @maintainer Stone White <stone@thone.io>
+#
+# @output MAIN1 Motor right
+# @output MAIN2 Motor left
+# @output MAIN3 Motor back
+# @output MAIN4 empty
+# @output MAIN5 Tilt servo right
+# @output MAIN6 Tilt servo left
+# @output MAIN7 Elevon right
+# @output MAIN8 Elevon left
+#
+# @board intel_aerofc-v1 exclude
+# @board bitcraze_crazyflie exclude
+#
+
+sh /etc/init.d/rc.vtol_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set CBRK_AIRSPD_CHK 162128
+
+	param set BAT_N_CELLS 4
+	param set BAT_A_PER_V 36.36
+	param set BAT_V_DIV  18.18
+
+	param set FW_ARSP_MODE 2
```

Comment:
`FW_ARSP_MODE` not have value 2.
 you can set to 1.
this pr will fix 13012 error: https://github.com/PX4/Firmware/pull/15394